tests/functional/reload/25-xtriggers.t

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+# THIS FILE IS PART OF THE CYLC WORKFLOW ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#-------------------------------------------------------------------------------
+
+# Ensure that xtriggers are preserved after reloads
+# See https://github.com/cylc/cylc-flow/issues/4866
+
+. "$(dirname "$0")/test_header"
+
+set_test_number 6
+
+init_workflow "${TEST_NAME_BASE}" <<'__FLOW_CONFIG__'
+[scheduling]
+    [[graph]]
+        R1 = """
+            broken
+            reload
+        """
+
+[runtime]
+    [[broken]]
+        script = false
+        # should be long enough for the reload to complete
+        # (annoyingly we can't make this event driven)
+        execution retry delays = PT1M
```

Comment:
Would it be worth moving this test into `tests/flakyfunctional` and possibly cutting down this delay, in order that it doesn't hog time in the `tests/functional` runs on GH Actions?

Reply:
I know, just conscious of the fact we are approaching the timeout limit on GH Actions. May have to either increase the limit or add an extra chunk soon